circle.yml

```diff
@@ -1,17 +1,26 @@
 machine:
+  environment:
+    YARN_VERSION: 0.17.8
+    PATH: "${PATH}:${HOME}/.yarn/bin"
   node:
     version: 6
 
 dependencies:
   pre:
-    - npm install -g yarn@0.16.1
+    - |
+      if [[ ! -e ~/.yarn/bin/yarn || $(yarn --version) != "${YARN_VERSION}" ]]; then
+        curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version $YARN_VERSION
+      fi
+  cache_directories:
+    - ~/.yarn
+    - ~/.yarn-cache
   override:
     - yarn install
 
 test:
   override:
     - yarn test
 
-general:
-  artifacts:
-    - "./coverage/lcov-report/"
+ general:
```

Comment:
This may be the reason of the CI failure. Try remove the extra whitespace? @bennett000